### PR TITLE
Fixes #7 - added script to deploy on gh-pages

### LINKED ISSUE #7: Figure out deploy strategy
Some fancy automated hook would be great. Depends on #6.

### PR DESCRIPTION
**Pull request by magsout**

### CODE REVIEW

**Comment by magsout** [comment]
Waiting for https://github.com/webcompat/webcompat-metrics-client/pull/48
- rebase
- push
- pray

**Comment by miketaylr** [comment]
> pray

👼

File: scripts/deploy.sh [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+siteSource="$1"
+
+if [ ! -d "$siteSource" ]
+then
+    echo "Usage: $0 <site source dir>"
+    exit 1
+fi
+
+# make a directory to put the gp-pages branch
```
Comment (miketaylr) [inline scripts/deploy.sh]: nit: gh-pages branch

File: scripts/deploy.sh [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+git init
+git remote add --fetch origin "$remote"
+
+# switch into the the gh-pages branch
+if git rev-parse --verify origin/gh-pages > /dev/null 2>&1
+then
+    git checkout gh-pages
+    # delete any old site as we are going to replace it
+    # Note: this explodes if there aren't any, so moving it here for now
+    git rm -rf .
```
Comment (miketaylr) [inline scripts/deploy.sh]: this looks scary! 😨 

I guess it will always be in `gh-pages-branch` directory...

File: scripts/deploy.sh [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+# make a directory to put the gp-pages branch
+mkdir gh-pages-branch
+cd gh-pages-branch
+# now lets setup a new repo so we can update the gh-pages branch
+git config --global user.email "$GH_EMAIL" > /dev/null 2>&1
+git config --global user.name "$GH_NAME" > /dev/null 2>&1
+git init
+git remote add --fetch origin "$remote"
+
+# switch into the the gh-pages branch
```
Comment (miketaylr) [inline scripts/deploy.sh]: nit: the the

File: scripts/deploy.sh [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    git checkout gh-pages
+    # delete any old site as we are going to replace it
+    # Note: this explodes if there aren't any, so moving it here for now
+    git rm -rf .
+else
+    git checkout --orphan gh-pages
+fi
+
+# copy over or recompile the new site
+cp -a "../${siteSource}/." .
```
Comment (miketaylr) [inline scripts/deploy.sh]: do we want a `-r` recursive flag here?

(maybe not needed if siteSource is flat?)

Comment (magsout) [reply scripts/deploy.sh]: for now it is.

**Comment by magsout** [comment]
@miketaylr 
>Assuming this works (you've tested?), lgtm

yes I'm, i'm using this script on my own business project. The new thing is about circleci, I discovered the v2 doc (using v1 doc)

**Pull request merged**